.vsts-ci.yml

```diff
@@ -162,60 +155,60 @@ extends:
+            # Note: These builds are also glibc like the glibc category, but that category uses containers, and doesn't publish zips and tarballs.
             - categoryName: Official
               publishArgument: $(_publishArgument)
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties)
+              osProperties: $(linuxOsglibcProperties)
               runTests: false
             - categoryName: Official
               targetArchitecture: arm
               runtimeIdentifier: linux-arm
               publishArgument: $(_publishArgument)
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties)
+              osProperties: $(linuxOsglibcProperties)
               runTests: false
             - categoryName: Official
               targetArchitecture: arm64
               runtimeIdentifier: linux-arm64
               publishArgument: $(_publishArgument)
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties)
+              osProperties: $(linuxOsglibcProperties)
               runTests: false
-            ### PORTABLE ###
-            - categoryName: Portable
+            ### glibc ###
+            - categoryName: glibc
               # Do not publish zips and tarballs. The linux-x64 binaries are already published by Official.
               publishArgument: $(_publishArgument) /p:PublishBinariesAndBadge=false
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties) /p:BuildSdkDeb=true
+              osProperties: $(linuxOsglibcProperties) /p:BuildSdkDeb=true
               runTests: false
-            - categoryName: Portable
+            - categoryName: glibc
               targetArchitecture: arm64
               runtimeIdentifier: linux-arm64
               # Do not publish zips and tarballs. The linux-arm64 binaries are already published by Official.
               publishArgument: $(_publishArgument) /p:PublishBinariesAndBadge=false
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties) /p:BuildSdkDeb=true
+              osProperties: $(linuxOsglibcProperties) /p:BuildSdkDeb=true
               runTests: false
-            - categoryName: Portable
-              container: centosStream9
+            - categoryName: glibc
+              container: azureLinux30Net10BuildAmd64
               # Do not publish zips and tarballs. The linux-x64 binaries are already published by Official.
               publishArgument: $(_publishArgument) /p:PublishBinariesAndBadge=false
               officialBuildProperties: $(_officialBuildProperties)
-              osProperties: $(linuxOsPortableProperties) /p:IsRPMBasedDistro=true
+              osProperties: $(linuxOsglibcProperties) /p:IsRPMBasedDistro=true
```

Comment:
I think now that the SDK official build is an assetless build, you can also get rid of the separate RPM and DEB jobs. They're already validated in the VMR and I don't believe you'll get much useful validation by having separate jobs for these in the official build.